### PR TITLE
docs: update README

### PR DESCRIPTION
**Pull request by SweeXordious**
## Overview

Closes https://github.com/celestiaorg/orchestrator-relayer/issues/29

## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

**Comment by SweeXordious** [comment]
The CI will fail for some links because they are added in the other PRs.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,62 @@
...
+```sh
+# Print help
+qgb --help
+```
+
+## How to run
+
+If you are a Celestia-app validator, all you need to do is run the orchestrator. Check [here](https://github.com/celestiaorg/orchestrator-relayer/blob/main/docs/orchestrator.md) for more details.
+
+If you want to run a relayer or target a new EVM chain, you will need to deploy a new QGB contract and run a relayer. Check [here](https://github.com/celestiaorg/orchestrator-relayer/blob/main/docs/relayer.md) for relayer docs and [here](https://github.com/celestiaorg/orchestrator-relayer/blob/main/docs/deploy.md) for how to deploy a new QGB contract.
```
Comment (evan-forbes) [inline README.md:30]:
```suggestion
If you want to post commitments on an EVM chain, you will need to deploy a new QGB contract and run a relayer. Check [here](https://github.com/celestiaorg/orchestrator-relayer/blob/main/docs/relayer.md) for relayer docs and [here](https://github.com/celestiaorg/orchestrator-relayer/blob/main/docs/deploy.md) for how to deploy a new QGB contract.
```

**Review by evan-forbes — DISMISSED** [review]
nice!

File: README.md (hunk 1 above)
Comment (evan-forbes) [reply README.md:30]: this felt a bit less repetitive and a bit more descriptive, but defifnitely leaving it as optional

**Review by evan-forbes — APPROVED** [review]

**Pull request merged**